docs/concepts/fundamentals/template_tag_syntax.md

```diff
@@ -30,14 +30,12 @@ so are still valid:
 </body>
 ```
 
-These can then be accessed inside `get_context_data` so:
+These can then be accessed inside `get_template_data` so:
 
 ```py
 @register("calendar")
 class Calendar(Component):
-    # Since # . @ - are not valid identifiers, we have to
```

Comment:
Removed this comment on kwargs with special characters, since that's not an issue with `get_template_data()`